src/index.ts

```diff
@@ -1,2 +1,4 @@
 export { TelemetryWrapper } from "./TelemetryWrapper";
-export { Session } from "./Session";
\ No newline at end of file
+export { Session } from "./Session";
```

Comment:
Seems we don't need this.

Reply:
It is used in the previous version, I'd rather not to remove this line.